example/src/main/kotlin/com/example/afterpay/data/CheckoutRequest.kt

```diff
@@ -9,4 +9,5 @@ data class CheckoutRequest(
     val email: String,
     val amount: String,
     val mode: CheckoutMode,
+    val isCashAppPay: Boolean? = null,
```

Comment:
I don't think we need the `null` value here. If the goal is to keep existing clients from having breaking changes, it seems safe to assume we could instead give it a default value of `false`?

Reply:
removed in [be751af](https://github.com/afterpay/sdk-android/pull/184/commits/be751af258677d4ca357fd49927c12316207c566)